model/src/cfd.rs

```diff
@@ -361,6 +361,19 @@ pub enum NoRolloverReason {
     Closed,
 }
 
+/// Reasons why we cannot collab close a CFD
+#[derive(thiserror::Error, Debug, PartialEq, Clone, Copy)]
+pub enum CannotSettleCollaborativeley {
```

Comment:
🔧 Should be `CannotSettleCollaboratively`.

Reply:
hahaha, I fixed it three times before opening the PR already 🙈